db/users/update.go

```diff
@@ -0,0 +1,70 @@
+package users
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	"github.com/Scalingo/go-scalingo/v6"
+	"github.com/Scalingo/go-utils/errors/v2"
+	"github.com/Scalingo/gopassword"
+)
+
+func UpdateUser(ctx context.Context, app, addonUUID, username string) error {
+	isSupported, err := doesDatabaseHandleUserManagement(ctx, app, addonUUID)
+	if err != nil {
+		return errors.Wrap(ctx, err, "get user management information")
+	}
+
+	if !isSupported {
+		io.Error(ErrDatabaseNotSupportUserManagement)
+		return nil
+	}
+
+	if usernameValidation, ok := isUsernameValid(username); !ok {
+		io.Error(usernameValidation)
+		return nil
+	}
+
+	password, confirmedPassword, err := askForPassword(ctx)
```

Comment:
suggestion: allow client to retry 3 times to set his own password

Reply:
Yep why not